core/src/main/java/org/elasticsearch/cluster/ClusterStateUpdateTask.java

```diff
@@ -75,4 +75,9 @@ public TimeValue timeout() {
     public Priority priority() {
         return priority;
     }
+
+    @Override
+    public final boolean runOnlyOnMaster() {
```

Comment:
can you add a comment that this is here to make it final as ClusterStateUpdateTask should only run on the master?